### PR TITLE
Make DEBUG=True on CI jobs

### PR DESCRIPTION
**Pull request by fao89**
[noissue]

### CODE REVIEW

File: templates/github/.github/workflows/scripts/before_install.sh.j2 [hunk 1]
```diff
@@ -166,6 +166,8 @@ then
   exit $s
 fi
 
+sed -i "s/DEBUG = False/DEBUG = True/" pulpcore/pulpcore/app/settings.py
```
Comment (mdellweg) [inline templates/github/.github/workflows/scripts/before_install.sh.j2:169]:
```suggestion
sed -i -e 's/DEBUG = False/DEBUG = True/' pulpcore/pulpcore/app/settings.py
```
Just a matter of personal style, but i like to explicit about the "expression" with sed. Also using single quotes usually keeps you from close to crazy shell escaping (a good habit even if not applicable here).

Comment (mdellweg) [reply templates/github/.github/workflows/scripts/before_install.sh.j2:169]: Needless to say, this is exactly what it is labelled: a suggestion

**Comment by daviddavis** [comment]
I'm curious about what benefit enabling DEBUG will bring? Extra logging?

**Comment by fao89** [comment]
> I'm curious about what benefit enabling DEBUG will bring? Extra logging?

yes, it helped me debug https://github.com/pulp/pulpcore/pull/1588

**Review by mdellweg — APPROVED** [review]

**Pull request merged**